format-specs/geoparquet.md

```diff
@@ -50,57 +50,160 @@ Version of the geoparquet spec used, currently 0.3.0
+| edges         | string              | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'.                                                                                                                                                                                                                                                                                                                             |
+| bbox          | \[number]           | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5).                                                                                                                                                                                                                                                                                                        |
+| epoch         | double              | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year.
 
 #### crs
 
 The Coordinate Reference System (CRS) is an optional parameter for each geometry column defined in geoparquet format.
 
-The CRS must be provided in [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_coordinate_reference_systems) version 2, also known as **WKT2**. WKT2 has several revisions, this specification only supports [WKT2_2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html).
+The CRS must be provided in [PROJJSON](https://proj.org/specifications/projjson.html) format, which is a JSON encoding of [WKT2:2019 / ISO-19162:2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html), which itself implements the model of [OGC Topic 2: Referencing by coordinates abstract specification / ISO-19111:2019](http://docs.opengeospatial.org/as/18-005r4/18-005r4.html).  Apart from the difference of encodings, the semantics is intended to be exactly the same as WKT2:2019, and PROJJSON can be morphed losslessly from/into WKT2:2019.
 
 If CRS is not provided, then all coordinates in the geometry must use longitude, latitude to store their data.
-If an implementation is CRS-aware and needs a CRS representation of the data it should assume a default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). It's equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude. The WKT2:2019 string for OGC:CRS84 is:
 
+If an implementation is CRS-aware and needs a CRS representation of the data it may assume a default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84), which is equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude.  However, there is no guarantee that coordinates
```

Comment:
Note: The "However..." can be removed if we instead state above that 

```
If CRS is not provided, then all coordinates in the geometry must use longitude, latitude based on the OGC:CRS84 CRS to store their data.
```

But without that, lacking a `crs` means that coordinates are in an unspecified CRS.

Reply:
I'm for removing the however and stating what it should be. Though perhaps we just say 'must use longitude, latitude with the WGS84 datum' to cover both OGC:CRS84 and EPSG:4326 (with axis order flipped).

Reply:
+1 on removing the "however ..". So that if CRS is not available, readers can assume WGS84